tests/lapack/full_matrix_17.cc

```diff
@@ -0,0 +1,66 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2005 - 2015 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// test LAPACKFullMatrix::reciprocal_condition_number() by comparing estimated
+// value to 1 / ( ||A||_1 * ||A^{-1}||_1 )
+
+#include "../tests.h"
+#include "create_matrix.h"
+#include <deal.II/lac/lapack_full_matrix.h>
+#include <deal.II/lac/full_matrix.h>
+#include <deal.II/lac/vector.h>
+
+#include <iostream>
+
+
+template <typename NumberType>
+void
+test(const unsigned int size)
+{
+  // Full matrix:
+  FullMatrix<NumberType> F(size), invF(size);
+  create_spd(F);
+  invF.invert(F);
+  const double l1 = F.l1_norm();
+  const double inv_l1 = invF.l1_norm();
+
+  // Lapack:
+  LAPACKFullMatrix<NumberType> M(size);
+  M = F;
+  M.set_property(LAPACKSupport::symmetric);
+  const double la_l1 = M.l1_norm();
+  M.compute_cholesky_factorization();
+  const double rcond = M.reciprocal_condition_number(la_l1);
+
+  deallog << 1./(l1*inv_l1) << " " << rcond << std::endl;
```

Comment:
this test is currently failing on the tester:
```
  DEAL::size=3
! DEAL::0.482025 0.563918
```
vs
```
  DEAL::size=3
! DEAL::0.434647 0.510929
```
so it looks like our random generator does not produce the same numbers here...

Reply:
my fault, i was not using the `Testing::rand()`, will fix...